SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1763,6 +1763,11 @@ void ActorSystem::RemoveActor(const Worker_EntityId EntityId)
 		}
 	}
 
+	if (Actor->GetTearOff())
```

Comment:
Conversely, we may well receive an EntityDelta::REMOVE at the same time as an updated TearOff value, which would not be caught in the previous control block as the channel is destroyed during TearOff handling.

Reply:
Given the length of this function and how many branches there seem to be it might be worth commenting this in code.